feat: introduce events (#63)
jexia/docs: docs/tools/README.md

</html>
```

## Events

> As we did try to handle "almost" everything for you, there are still cases where you like to act on actions/events when something happen.

Reviewer: ```suggestion
As we try to handle "almost" everything for you, there are still cases where you need to act on actions/events when something happens.
```

> There for you can use the event system of the SDK, to do some side effects on a login for example.

Reviewer: ```suggestion
Therefore you can use the event system of the SDK, for example to execute custom code on a login.
```
